### PR TITLE
Update video-js styling to be responsive.

### LINKED ISSUE #666: Evaluate responsive nature of VideoJS player
Possibly look into the `fluid: true` option to VideoJS: https://coolestguidesontheplanet.com/videodrome/videojs/

See #672 for an issue on small devices.

### PR DESCRIPTION
**Pull request by jkeck**
Closes #666 :metal: 

responsive-video

### CODE REVIEW

File: app/assets/stylesheets/media.scss [hunk 1]
```diff
@@ -101,3 +102,32 @@
   }
 
 }
+
+// VideoJS CSS overrides for responsive layout
+// See http://andreassauer.name/experimente/video-js/demo.html
+.video-js {
+  width: 100% !important;
```
Comment (jkeck) [inline app/assets/stylesheets/media.scss]: Given that we're overriding the upstream css, this seems to be required (as indicated by the link mentioned in the comment).

**Comment by ndushay** [comment]
LGTM;  did you look at this in FF, Chrome and Safari?  Or should I spin it up?

**Comment by jkeck** [comment]
I briefly checked my installed browsers, but please feel free to take a look.  This branch is what is currently being embedded on purl-test, so feel free to check there (so you don't have to spit it up yourself and get it to play video).

**Comment by ndushay** [comment]
👍   for FF and Safari  (b/c I know you work in Chrome, I didn't check it)

**Comment by tingulfsen** [comment]
I'm normally not cool with the use of `!important`, but in this case I think it's OK to force a very specific and localized behavior.

**Pull request merged**